source/robot_model/include/robot_model/Model.hpp

```diff
@@ -206,14 +219,20 @@ inline void Model::set_urdf_path(const std::string& urdf_path) {
   this->urdf_path_->set_value(urdf_path);
 }
 
-inline unsigned int Model::get_nb_joints() const {
+inline unsigned int Model::get_number_of_joints() const {
   // subtract 1 because of the 'universe' joint
-  return this->robot_model_.njoints - 1;
+  return this->robot_model_.nq;
+}
+
+inline std::vector<std::string> Model::get_joint_frames() const {
+  // model contains a first joint called universe that needs to be discarded
```

Comment:
Is the first joint called `universe` or `root_joint`? not that it matters too much for this comment. I appreciate that you are showing that the +1 is intentional

Reply:
Its called `universe`